apps/fxc-front/public/static/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "short_name": "FlyXC",
+    "name": "FlyXC: GPS track visualization, flight planning, live tracking, and more",
+    "icons": [
+        {
+            "src": "/static/img/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
+    "start_url": "https://flyxc.app/?source=pwa",
```

Comment:
I don't think we need `?source=pwa` ?

Reply:
I cargo culted that from web.dev, but definitely unnecessary (except for analytics etc).  Removed now.